src/main/java/com/commercetools/sync/products/helpers/ProductReferenceResolver.java

```diff
@@ -184,35 +202,30 @@ public ProductReferenceResolver(@Nonnull final ProductSyncOptions productSyncOpt
      * in the CTP project, the resultant draft would remain exactly the same as the passed product draft
```

Comment:
Please update the Javadoc that its using draft builder now instead.

Reply:
51ead5e4